CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.1]
+
+### Fixed
+* Updated admonition types to those that are currently supported by `mkdocs-material`.
```

Comment:
```suggestion
* Updated admonition types to those that are supported by `mkdocs-material` v9.4.2+.
```
I didn't think about this on the dev PR, but is it worth specifying the mkdocs version?